apps/web/src/components/milestone-logs/recruitment/Milestone.tsx

```diff
@@ -120,60 +150,68 @@ export const EditMilestone: React.FC<EditMilestoneProps> = props => {
       </button>
     );
   };
-  return (
-    <>
-      {editing !== milestone ? (
-        <div className='border border-gray-200 rounded bg-bcLightGray my-2 p-5'>
-          <div className='w-full'>
-            <div className='flex items-center font-bold text-black '>
-              <span className='capitalize'>{milestone.status.status}</span>
-              <span className='mx-2'>|</span>
-              <span className='mr-2'>
-                <img src={calendarIcon.src} alt='calendar' width={16} height={16} />
-              </span>
-              <span>{formatDate(milestone.start_date)}</span>
-              {(milestone.updated_by?.email || milestone.added_by?.email) && (
-                <>
-                  <span className='mx-2'>|</span>
-                  <span className='mr-2'>
-                    <img src={userIcon.src} alt='user' />
-                  </span>
-                  <span>Last updated by</span>
-                  <a
-                    className='ml-2'
-                    href={`mailto: ${milestone.updated_by?.email || milestone.added_by?.email}`}
-                  >
-                    {milestone.updated_by?.email || milestone.added_by?.email}
-                  </a>
-                </>
-              )}
-              <button
-                className='ml-auto mr-2'
-                onClick={() => onEditing(milestone)}
-                disabled={!!editing && milestone === editing}
-              >
-                <img src={editIcon.src} alt='edit milestone' />
-              </button>
-              {deleteButton()}
-            </div>
-            <span className='text-sm text-black break-words'>
-              {milestone.notes || 'No Notes Added'}
+
+  const isRecruitmentRelated = () => {
```

Comment:
method name hints it will return boolean.

Reply:
yep good catch, will change fn name